.changeset/ten-keys-begin.md

```diff
@@ -0,0 +1,10 @@
+---
+"@patternfly/pfe-tile": major
+---
+
+Implements `<pfe-tile>` aligned with [PatternFly v4](https://patternfly.org/components/tile)
+
+
+- Adds BaseTile and pfe-tile
+- Added `selected` attribute to highlight the tile
```

Comment:
these two lines should be snippets or in the docs instead of "added" - everything here is "added", right?